src/paymentservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraClientImplTest.java

```diff
@@ -0,0 +1,31 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
+
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.exception.PaymentServiceException;
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types.SinglePaymentRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class BamboraClientImplTest {
+
+    private BamboraClientImpl sut;
+
+    @Test
+    public void withValidParamShouldReturnAUrl() throws PaymentServiceException, MalformedURLException {
+
+
+        sut = new BamboraClientImpl(new URL("https://test.com/payment.asp"), "1", "12", 5);
```

Comment:
remind me what sut stands for again please

Reply:
System Under Test good question :-)

Reply:
thank you sir